src/Table/HeaderCell.jsx

```diff
@@ -25,14 +26,19 @@ export default function HeaderCell({
         sortable && cssClass.SORTABLE,
         className
       )}
-      noWrap
       onClick={() => sortable && onSortChange()}
       width={width}
     >
-      <div className={cssClass.LABEL}>
-        {children}
-      </div>
-      {sortable && <SortIcons direction={activeSortDirection} className={cssClass.SORT} />}
+      <FlexBox alignItems={ItemAlign.CENTER} >
```

Comment:
yeah, if the label is the only thing that needs to wrap, that can separately be changed to `white-space: normal` (while the cell stays `nowrap`)